app/controllers/tasks_controller.rb

```diff
@@ -92,13 +92,15 @@ def enforce_policy
   end
 
   def notify_task_updated!
-    if @task_completed
+    if @task_completion_change
+      action = task.completed? ? 'complete' : 'incomplete'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.